.markdownlint.yaml

```diff
@@ -0,0 +1,14 @@
+# See https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md
+# and https://github.com/DavidAnson/markdownlint/blob/main/README.md
+
+# Default state for all rules
```

Comment:
```suggestion
# This should be kept in sync with https://github.com/open-telemetry/opentelemetry-specification/blob/main/.markdownlint.yaml

# Default state for all rules
```
Maybe add a comment like this since the goal is to provide Markdown that is valid for the spec repo and hence use that in the test files.